src/applications/vaos/referral-appointments/redux/reducers.js

```diff
@@ -103,6 +106,18 @@ function ccAppointmentReducer(state = initialState, action) {
         ...state,
         referralFetchStatus: FETCH_STATUS.failed,
       };
+    case SET_SELECTED_SLOT:
+      return {
+        ...state,
+        selectedSlot: action.payload,
+      };
+    case SET_INIT_REFERRAL_FLOW:
+      return {
+        ...state,
+        selectedProvider: {},
+        providerFetchStatus: FETCH_STATUS.notStarted,
+        selectedSlot: null,
```

Comment:
Should this be empty string to match initial state?

Reply:
Yes! I'll update that

Reply:
@wullaski updated